### ISSUE TITLE
A list of prereqs could be handy

### ISSUE THREAD

**Issue by dsmilkov** [tensorflow/tfjs#57]
_From @sgserg on April 2, 2018 17:16_

Examples work neither on OSX 10.9 (errors out on unsupported node version)
nor in iOS 11:

![beb1db23-6dcf-4596-b6cb-7a8966edc864](https://user-images.githubusercontent.com/5439847/38206449-2058728c-36b3-11e8-965c-f8b599a7fe50.jpeg)

_Copied from original issue: tensorflow/tfjs-examples#49_

**Comment by dsmilkov** [comment]
Hi @sgserg,

Can you share more info about the error you are seeing? Thanks!

**Comment by tafsiri** [comment]
Also, if you are trying to run the examples locally you should have node version 8.9.0 or greater. This is currently documented in the package.json files for each project.

**Comment by tafsiri** [comment]
@sgserg You could look at instructions at https://nodejs.org/en/ for how to get a more recent node.js version installed. 

To your question about examples on iOS it could be that your device doesn't support webgl float textures, this results in lower precision and may need more training time in order to converge. The red line that renders shows that the example runs, it just didn't learn the function effectively. Once you get it running locally, you can try tweaking the number of iterations and learning rate to see how that affects performance on that device. I've found that on a recent iphone the model learns the correct parameters fairly quickly.
